### DISCUSSION TITLE
Add the Adaptations Programming Guide to the Wiki?

### DISCUSSION THREAD

**Discussion by Andy2No** [christofmuc/KnobKraft-orm#181] [Ideas]
The main Wiki page is still looking a bit sparse, and it still takes me a while to find the link to the adaptations programming guide.  Maybe a link to it should be on that first Wiki page?

https://github.com/christofmuc/KnobKraft-orm/blob/master/adaptions/Adaptation%20Programming%20Guide.md

**Comment by christofmuc** [comment]
@Andy2No Good idea, just did it. There could certainly be more info in the Wiki, but the programming guide itself is fairly up to date I pride myself to say.

**Reply by Andy2No** [reply]
Cool.  Thanks.

**Comment by christofmuc** [comment]
Closing this as done.
